package.json

```diff
@@ -35,6 +35,7 @@
   "private": false,
   "dependencies": {
     "@angular/animations": "^11.1.0",
+    "@angular/cdk": "^11.2.13",
```

Comment:
Just wondering if we should add a short one-liner to the contribution docs 🤔. Just so people know that the cdk resources are available to play with.

Reply:
I've added:
```
# Angular Material CDK
The `@angular/cdk` can be used as long as only the necessary functionality is imported. 
```